src/core_seaice/model_forward/mpas_seaice_core_interface.F

```diff
@@ -442,6 +452,9 @@ subroutine setup_packages_column_physics(configPool, packagePool, ierr)!{{{
      pkgTracerZAerosolsActive            = config_use_zaerosols
      pkgTracerZSalinityActive            = config_use_vertical_zsalinity
 
+     pkgColumnTracerSnowDensityActive     = config_use_effective_snow_density
```

Comment:
Change pkgColumnTracerSnowDensityActive to pkgColumnTracerEffectiveSnowDensityActive